backend/app/core/game_config.py

```
@@ -640,6 +641,32 @@ class GameConfig(BaseSettings):
     exploration: ExplorationConfig = Field(default_factory=ExplorationConfig)
 
 
+# Medical room production mapping (room name lowercase → product type)
+# Used to replace fragile string matching like "medbay" in room.name.lower()
+MEDICAL_ROOM_PRODUCTION: dict[str, str] = {
+    "medbay": "stimpak",
+    "science lab": "radaway",
+}
+
+
+def compute_medical_capacity(rooms: Sequence["Room"]) -> dict[str, int]:
```

Ruff (0.15.17)

[error] Undefined name `Room`

(F821)